Sources/Draw/GLRenderer.cpp

```diff
@@ -256,6 +256,9 @@ namespace spades {
 			imageManager = NULL;
 			delete fbManager;
 			fbManager = NULL;
+			
+			delete device;
```

Comment:
Thank you for noticing! Actually, however, this is not necessary because `device` has a type of `Handle<IGLDevice>` which is a smart pointer, and therefore it's released automatically when `GLRenderer` is destroyed. (or maybe I'm missing something...?)